wandb/jupyter.py

```diff
@@ -66,7 +65,7 @@ def __init__(self, path=None, opts=None):
 
     def maybe_display(self) -> bool:
         if not self.displayed and (self.path or wandb.run):
-            display(self)
+            IPython.display.display(self)
```

Comment:
what version did they add this? trying to understand why we were using core... instead of this

Reply:
It's existed since at least 4.x (released in 2015) https://ipython.readthedocs.io/en/4.x/api/generated/IPython.display.html#IPython.display.display

Reply:
lol... anyways... this is much better, just want to make sure!